Config/Inc/config.h

```diff
@@ -36,24 +41,70 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 
 //--------------------------------------------------------------------------------
 // MAX and MIN limits of lithium ion cells
+#ifdef CUSTOM_MIN_VOLT_CHARGE
+#define MIN_VOLTAGE_CHARGING_LIMIT CUSTOM_MIN_VOLT_CHARGE
+#else
 #define MIN_VOLTAGE_CHARGING_LIMIT      2550        // Under voltage limit for charging (milliVolts) 
+#endif
+
+#ifdef CUSTOM_MIN_VOLT
+#define MIN_VOLTAGE_LIMIT CUSTOM_MIN_VOLT
+#else
 #define MIN_VOLTAGE_LIMIT				2700		// Under voltage limit (milliVolts)	(actual min: 2.5V)
+#endif
+
+#ifdef CUSTOM_MAX_VOLT
+#define MAX_VOLTAGE_LIMIT CUSTOM_MAX_VOLT
+#else
 #define MAX_VOLTAGE_LIMIT				4100		// Over voltage limit (milliVolts)		(actual max: 4.2V)
+#endif
+
+#ifdef CUSTOM_CHARGE_DIS_VOLT
+#define CHARGE_DISABLE_VOLTAGE CUSTOM_CHARGE_DIS_VOLT
+#else
 #define CHARGE_DISABLE_VOLTAGE          4000        // Voltage to stop charging at
+#endif
 
 // make sure we don't enable charging if we're too close to the voltage limit
 #if MAX_VOLTAGE_LIMIT - 100 < CHARGE_DISABLE_VOLTAGE
 #error "Charging maximum voltage is too close to voltage trip limit!"
 #endif
 
+#ifdef CUSTOM_MAX_DISCHARGE_TEMP
+#define MAX_DISCHARGE_TEMPERATURE_LIMIT CUSTOM_MAX_DISCHARGE_TEMP
```

Comment:
@manthanand some of these names are also a bit verbose. `MAX` and `LIMIT` seem redundant together, for example. Might be annoying to type out on the command line

Reply:
I agree. but my assumption is that the verbose command that has to be typed out is "CUSTOM_whatever". I think that parameter should be changed to something easier to deal with. It would also be nice if all parameters were changed too be smaller since some of it is redundant as well.

Reply:
Good point. You could give them the same name with something like
```C
#ifndef MAX_DISCHARGE_TEMP
#define MAX_DISCHARGE_TEMP 60000
#endif
```

Reply:
Yeah, I'll change them to use same name